### PR TITLE
New navigation

### PR DESCRIPTION
**Pull request by rfraposa**

### CODE REVIEW

**Review by mshustov — CHANGES_REQUESTED** [review]
Does the branch contain the recent changes from `main` branch? Kafka HTTP Sink and Java Client docs have already been refactored.

**Comment by rfraposa** [comment]
@mshustov - yes, it should. I just created this branch yesterday, and I update it with `main` all the time.

File: docs/en/integrations/data-ingestion/kafka/cloud/confluent/kafka-connect-http.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    * `Batch batch size` - Per ClickHouse recommendations, set this to at least 1000.
+* HTTP server authentication
+    * `Endpoint Authentication type` - BASIC
+    * `Auth username` - ClickHouse username
+    * `Auth password` - ClickHouse password
+* HTTP server retries - Settings here can be adjusted according to requirements. Timeouts specifically may need adjusting depending on latency.
+    * `Retry on HTTP codes` - 400-500 but adapt as required e.g. this may change if you have an HTTP proxy in front of ClickHouse.
+    * `Maximum Reties` - the default (10) is appropriate but feel to adjust for more robust retries.
+
+<img src={require('./images/http_sink_config.png').default} class="image" alt="Connecting Confluent HTTP Sink" style={{width: '50%'}}/>
```
Comment (mshustov) [inline docs/en/integrations/data-ingestion/kafka/cloud/confluent/kafka-connect-http.md:139]: The file was removed in https://github.com/ClickHouse/clickhouse-docs/pull/770/ 3 weeks ago

File: docs/en/integrations/data-ingestion/kafka/cloud/confluent/confluent-intro.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (mshustov) [inline docs/en/integrations/data-ingestion/kafka/cloud/confluent/confluent-intro.md:1]: You removed this file in https://github.com/ClickHouse/clickhouse-docs/pull/767

**Review by mshustov — COMMENTED** [review]
@rfraposa I added a couple of examples of missing changes

**Comment by rfraposa** [comment]
I might need some Github help then. I don't understand why this branch is out of sync with `main`.

**Comment by mshustov** [comment]
@rfraposa, how did you create https://github.com/ClickHouse/clickhouse-docs/pull/850/commits/814eda1a46961105a686aba8d92296bd6276c331 commit?

**Comment by rfraposa** [comment]
I created the `new-nav` branch from `main`, then copied-and-pasted a bunch of changes into that folder structure, and then pushed it into the `new-nav` branch.

FWIW: I would have thought my branch would have a conflict with `main` if I ever tried to merge it, but I don't see any conflicts - even though it clearly must have some.

File: docs/en/integrations/data-ingestion/kafka/cloud/confluent/kafka-connect-http.md (hunk 1 above)
Comment (rfraposa) [reply docs/en/integrations/data-ingestion/kafka/cloud/confluent/kafka-connect-http.md:139]: I figured it out. I accidentally re-added these Kafka files (bad copy-and-pasting). Thanks for noticing!

**Comment by mshustov** [comment]
> I created the new-nav branch from main, then copied-and-pasted a bunch of changes into that folder structure, and then pushed it into the new-nav branch.

There is no conflict if you `git add` already removed content because, well, `main` branch doesn't have it. Could you remove unnecessary changes from the commit you created manually?

**Pull request merged**